src/models/orders.rs

```diff
@@ -26,6 +26,20 @@ pub struct Order {
 }
 
 impl Order {
+    pub fn get_current_sell_amount(&self) -> U256 {
+        if let Some(current_sell_amount) = self.current_sell_amount {
+            current_sell_amount
+        } else {
+            self.sell_amount
+        }
+    }
```

Comment:
ubernit: Could you add some spacing to the methods, I find its harder to read without it:
```suggestion
    }

```